deploy/templates/gcs-manifests/gcs-gd2.yml.j2

```diff
@@ -61,37 +71,7 @@ spec:
         - name: gluster-kmods
           hostPath:
             path: "/usr/lib/modules"
-
----
-kind: Service
-apiVersion: v1
-metadata:
-  name: glusterd2-client
-  namespace: {{ gcs_namespace }}
-  labels:
-    gcs: glusterd2-service
-spec:
-  selector:
-    gcs: glusterd2
-  ports:
-    - protocol: TCP
-      port: 24007
-      targetPort: 24007
-
----
-kind: Service
-apiVersion: v1
-metadata:
-  name: glusterd2-client-nodeport
-  namespace: {{ gcs_namespace }}
-  labels:
-    gcs: glusterd2-service
-spec:
-  selector:
-    gcs: glusterd2
-  ports:
-    - protocol: TCP
-      port: 24007
-      targetPort: 24007
-      nodePort: 31007
-  type: NodePort
+        - name: glusterd2-statedir
+          emptyDir: {}
```

Comment:
For now, as @jarrpa suggested, I'd recommend hostPath here & for logdir.

Reply:
I'll do this. Is there a preferred host path for this

Reply:
For now, just go with /var/lib/glusterd2. I'm still trying to figure out something better.

It's going to eventually need to move w/ the pod. Options I can think of are:
- Use a PV for it (This can get wasteful in both EBS and local PV scenarios)
- Have an init container partition a data PV/device to make room for the state dir
- Use a Secret (I think a pod can modify this)
- Use a ConfigMap (This can't be written by the pod)

Reply:
I agree, just use the same paths that would be expected on the host and mount them to the same paths in the container.

As a note, Secrets can't be modified from within the container, just like ConfigMaps.